src/azure-cli/HISTORY.rst

```diff
@@ -56,6 +56,15 @@ Release History
 
 * Fix issue in az backup policy list-associated-items. Added optional BackupManagementType parameter.
 
+**BotService**
```

Comment:
Please rebase dev branch and make your release note available for next release.

Reply:
[Done](https://github.com/Azure/azure-cli/pull/11318/files#diff-364518622fafca75168eac23785eaca6R3-R14)!